src/core/Transport.spec.js

```diff
@@ -1,19 +1,85 @@
 import sinon from 'sinon';
 import Transport from './Transport.js';
+import {TARGETED_DATA, TARGET_SEARCH_MODES} from './constants.js';
 
 describe('Transport Unit Tests', () => {
+
   const tenancyId = '123-456';
+
+  const validTarget = {
+    targetedData: TARGETED_DATA.FULL_URL,
+    searchMode: TARGET_SEARCH_MODES.PARTIAL_MATCH,
+    searchValue: 'example.com:8232'
+  };
+
+  const validTargetWithRegex = {
+    targetedData: TARGETED_DATA.URL_PATHNAME,
+    searchMode: TARGET_SEARCH_MODES.REGEX,
+    searchValue: '.+(\/test\/).+'
+  };
+
+  const invalidTarget = {
+    targetedData: TARGETED_DATA.FULL_URL,
+    searchMode: 'gibberish',
+    searchValue: 'localhost:3000'
+  };
+
+  const incompleteTarget = {
+    targetedData: TARGETED_DATA.FULL_URL,
+    searchValue: 'localhost:3000'
+  };
```

Comment:
@robertdumitrescu this only covers missing searchMode, can we add coverage for missing targetedData and searchValue?